app-header/app-header.html

```diff
@@ -284,6 +301,25 @@
       },
 
       /**
+       * Returns a  a reference to the element that remains visible in condensed mode.
+       *
+       * @return {HTMLElement}?
+       */
+      _getTopBar: function() {
+        var topBar;
+        Polymer.dom(this.$.content).getDistributedNodes().forEach(function(node) {
+          if (node.nodeType === Node.ELEMENT_NODE) {
+            if (node.hasAttribute('primary')) {
+              topBar = node;
+            } else if (!topBar) {
+              topBar = node;
+            }
+          }
+        }, this);
+        return topBar;
```

Comment:
Consider enhancing the code here so that it exits the loop once the topBar is found.  We can do that later in another PR if you wish.